### ISSUE TITLE
dollar-variable-first-in-block - ignore @use

### ISSUE THREAD

**Issue by EvgenyOrekhov** [kristerkari/stylelint-scss#517]
I think `"ignore": ["imports"]` option should also ignore `@use`.

Real-world example:

```scss
@use "sass:color";

$primary-color: #f26e21 !default;
$secondary-color: color.change($primary-color, $alpha: 0.08) !default;
```

Actual: `Expected $-variable to be first in block (scss/dollar-variable-first-in-block)`

Expected: no warnings

**Comment by kristerkari** [comment]
I was looking at the other open issues, and this was a suggestion that came up there too, so I agree that the imports option could support `@use` and `@forward` too.
